pkg/kubectl/cmd/attach.go

```diff
@@ -194,10 +194,37 @@ func (p *AttachOptions) Run() error {
 	}
 	t.Raw = tty
 
-	fn := func() error {
-		if tty {
-			fmt.Fprintln(p.Out, "\nHit enter for command prompt")
+	var resizeChan chan term.Size
```

Comment:
Feel like this could be abstracted out of this method (realize this is still WIP), but doesn't feel cleanly coupled to the rest of the method.